From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Luciano Gomes <luciano.dittgen@ossystems.com.br>
Date: Tue, 16 Jun 2026 00:00:00 +0000
Subject: [PATCH] params: disable num default features

params only uses num::NumCast, so it does not need num's default
bigint, complex, rational, and rustc-serialize feature set. Disabling
the defaults avoids compiling legacy rustc-serialize derives that no
longer build with current Rust.

Upstream-Status: Inappropriate [oe-specific]

Signed-off-by: Luciano Gomes <luciano.dittgen@ossystems.com.br>
---
 Cargo.toml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Cargo.toml b/Cargo.toml
index b7f1c63..e648f9c 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -25,6 +25,7 @@ version = "0.8"
 
 [dependencies.num]
 version = "0.1"
+default-features = false
 
 [dependencies.tempdir]
 version = "0.3"
-- 
2.43.0
